### PR TITLE
pkgapu.sh: fix mkimage path

### PR DESCRIPTION
**Pull request by houlz0507**
#### Problem solved by the commit
Incorrect env variable dependency.

### CODE REVIEW

File: src/runtime_src/tools/scripts/pkgapu.sh [hunk 1]
```diff
@@ -27,10 +27,12 @@ usage()
 	echo "  options:"
 	echo "          -help                           Print this usage"
 	echo "          -images                         Versal images path"
-	echo "          -petalinux                      Petalinux path"
 	echo "          -clean                          Remove build files"
         echo "          -output                         output path"
-	echo ""
+	echo "This script requires tools: mkimage, xclbinutil, bootgen, rpmbuild, dpkg-deb. "
```
Comment (rozumx) [inline src/runtime_src/tools/scripts/pkgapu.sh:32]: You might wish to check to see if the XILINX_XRT and XILINX_VITIS environment variables are set OR do a check to see if these applications are found with the `which` command.

**Review by rozumx — APPROVED** [review]
Approved with comments.

File: src/runtime_src/tools/scripts/pkgapu.sh (hunk 1 above)
Comment (houlz0507) [reply src/runtime_src/tools/scripts/pkgapu.sh:32]: Added which command to check tools.

File: src/runtime_src/tools/scripts/pkgapu.sh [hunk 2]
```diff
@@ -154,8 +156,8 @@ if [[ ! -d $IMAGES_DIR ]]; then
 fi
 IMAGES_DIR=`realpath $IMAGES_DIR`
 
-if [ -f $SETTINGS_FILE ]; then
-	source $SETTINGS_FILE
+if [[ ! (`which mkimage` && `which bootgen` && `which xclbinutil`) ]]; then
+	error "Please source Xilinx VITIS and Petalinux tools to make sure mkimage, bootgen and xclbinutil is accessible."
```
Comment (rbramand-xilinx) [inline src/runtime_src/tools/scripts/pkgapu.sh:160]: @houlz0507 sourcing petalinux will not set mkimage path in PATH variable, so mkimage will not be picked automatically. That is the reason why I have hardcoded mkimage path earlier.

so users of this script can either install mkimage using **yum install -y uboot-tools** in centos/rhel machine to get mkimage binary or add petalinux mkimage path to PATH variable to autopick mkimage.

should we add setting of PATH variable in error message or it is okay?

**Review by rbramand-xilinx — APPROVED** [review]

File: src/runtime_src/tools/scripts/pkgapu.sh (hunk 2 above)
Comment (houlz0507) [reply src/runtime_src/tools/scripts/pkgapu.sh:160]: The mkimage path is shown in error message
Usage:  [options]
  options:
          -help                           Print this usage
          -images                         Versal images path
          -clean                          Remove build files
          -output                         output path
This script requires tools: mkimage, xclbinutil, bootgen, rpmbuild, dpkg-deb. 
There is mkimage in petalinux build, e.g.
/proj/petalinux/2021.2/petalinux-v2021.2_daily_latest/tool/petalinux-v2021.2-final/components/yocto/buildtools/sysroots/x86_64-petalinux-linux/usr/bin/mkimage

Comment (rbramand-xilinx) [reply src/runtime_src/tools/scripts/pkgapu.sh:160]: looks good, I will update the BAI with this change. Thanks.

**Review by stsoe — APPROVED** [review]

**Pull request merged**